### PR TITLE
new uplink items depending on your race

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
this pr currently has a few unique items for a couple of our round start races that they can buy in there uplink
angel potion for human 7tc
fire breath for lizards 7tc
killer tomato seeds for podpeople 5tc
radiation grenades for plasmamen 2 tc
#### Changelog

:cl:  
rscadd: added a few new items to the syndicate uplink so we have some cool items that different races can experiment with
wip: This is work in progress  
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
My god look at all those commits

**Comment by boodaliboo** [comment]
> My god look at all those commits
these were all webedits i made during class so i just made a seperate pr for each thing

File: yogstation/code/game/objects/items/premadepapers.dm [hunk 1]
```diff
@@ -154,6 +154,13 @@
 	<b>Signed, </b><span class=\"paper_field\"></span><br><br><br>
 	<b>Stamp Below if Approved</b>"}
 
+/obj/item/paper/paperwork/kojuhs_rant
+	name = "The Super SECRET PRETERNIS GUIDE"
+	fields = 2
+	info = {"<center><h3>THE RANT</h3></center><hr>
+	<b>powergame bad: </b><span class=\"paper_field\"></span><br><br>
+	<b>-kojuh: </b><span class=\"paper_field\"></span><br><br>
```
Comment (alexkar598) [inline yogstation/code/game/objects/items/premadepapers.dm]:
```suggestion
	info = {"<center><h3>THE RANT</h3></center><hr>\
	<b>powergame bad: </b><span class=\"paper_field\"></span><br><br>\
	<b>-kojuh: </b><span class=\"paper_field\"></span><br><br>"
```

File: code/game/objects/items/dna_injector.dm [hunk 2]
```diff
@@ -164,6 +164,11 @@
 	name = "\improper DNA injector (Stutt.)"
 	desc = "Makes you s-s-stuttterrr."
 	add_mutations = list(NERVOUS)
+	
+	obj/item/dnainjector/firebreath
```
Comment (alexkar598) [inline code/game/objects/items/dna_injector.dm]:
```suggestion
	/obj/item/dnainjector/firebreath
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: yogstation/code/game/objects/items/premadepapers.dm (hunk 1 above)
Comment (alexkar598) [reply yogstation/code/game/objects/items/premadepapers.dm]: Github is being an ass today,be sure to do it properly not weirdly like this

**Comment by fluffe9911** [comment]
racist

**Comment by alexkar598** [comment]
Note: I will be closing this PR on the 24h mark,its just shit

**Comment by boodaliboo** [comment]
> Note: I will be closing this PR on the 24h mark,its just shit
why i spent a few hours on it

**Comment by boodaliboo** [comment]
> Note: I will be closing this PR on the 24h mark,its just shit

and it passed travis

**Comment by Nickvr628** [comment]
I don't think this is a shit PR, it is a bit memey and might end up needing a balance pass later on but I think it is fine.

**Comment by boodaliboo** [comment]
> Note: I will be closing this PR on the 24h mark,its just shit

alex changed his mind after i edited this

**Comment by MegaEmpirical** [comment]
Racist. No plasmamen nor moth love.

File: code/game/objects/items/grenades/chem_grenade.dm [hunk 3]
```diff
@@ -324,6 +324,24 @@
 	beakers += B1
 	beakers += B2
 
+/obj/item/grenade/chem_grenade/radiation
+	name = "Rad Bomb"
+	desc = "the best grenade to irridiate the fuck out of someone"
+	stage = GRENADE_READY
+
+/obj/item/grenade/chem_grenade/metalfoam/Initialize()
```
Comment (alexkar598) [inline code/game/objects/items/grenades/chem_grenade.dm]: epic
```suggestion
/obj/item/grenade/chem_grenade/radiation/Initialize()
```

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 4]
```diff
@@ -104,7 +104,7 @@
 
 /obj/item/reagent_containers/food/snacks/deadmouse
 	name = "dead mouse"
-	desc = "It looks like somebody dropped the bass on it. A lizard's favorite meal."
+	desc = "It looks like somebody dropped the bass on it. A Degenerate's favorite meal."
```
Comment (alexkar598) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: no

File: code/modules/uplink/uplink_items.dm [hunk 5]
```diff
@@ -1603,6 +1603,34 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	name = "Angel Potion"
+	desc = "We mixed a bird and a human and we somehow made a potion that turns you into a holy creatures"
+	cost = 7
+	item = /obj/item/reagent_containers/glass/bottle/potion/syndicate
+	restricted_species = list("human")
+	
+/datum/uplink_item/race_restricted/killertomatos
+	name = "Killer Tomatos"
+	desc = "The Syndicates local gardeners brewed these up for our plant comrades (does not work against fellow plants)."
+	cost = 5
```
Comment (alexkar598) [inline code/modules/uplink/uplink_items.dm]: too high for something you need to grow

File: code/modules/uplink/uplink_items.dm [hunk 6]
```diff
@@ -1603,6 +1603,34 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	item = /obj/item/seeds/tomato/killer
+	restricted_species = list("pod")
+
+/datum/uplink_item/race_restricted/radiationbomb
+	name = "Radiation grenade"
+	desc = "A radiation bomb guarenteed to irridiate the fuck out of non gaseous lifeforms."
+	cost = 2
```
Comment (alexkar598) [inline code/modules/uplink/uplink_items.dm]: too low for an AOE ur fucked grenade

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
I’ll do your suggestions In a bit thanks

**Review by alexkar598 — APPROVED** [review]

**Comment by neeshacark** [comment]
Still no tail eswords for lizards

**Pull request merged**